scheduler/docs/configuration.asc

```diff
@@ -27,6 +27,9 @@ The next several sections will document what every option does.
 `:port`::
   This is the port that the REST API will bind to.
 
+`:hostname`::
+  This is the hostname of the server Cook is running on.
+
```

Comment:
In waiter, we default this to "localhost" (https://github.com/twosigma/waiter/blob/4f6147d888fce6aecda3825c5698a6400a1ddc4e/waiter/src/waiter/settings.clj#L195); should we do the same here?

Reply:
Done